mbed_tools_ci/tag_and_release.py

```diff
@@ -39,10 +43,33 @@ def tag_and_release(mode: CommitType) -> None:
     _release_to_pypi()
 
 
+def _remove_old_documentation(git: GitWrapper) -> None:
+    """Remove existing docs first to make sure no stale pages left behind."""
+    with cd(ROOT):
+        subprocess.check_call(['rm', '-r', DOCS_DIR])
+        subprocess.check_call(['mkdir', DOCS_DIR])
```

Comment:
I think there might be a better way of doing this. Ideally I would do `rm -r docs/*` but I got an interactive prompt, even when I included `interactive=never`.

Reply:
That sounds good - I will take a look!

Reply:
Yeah I think using `pathlib` or `shutil` is better than calling `rm` and `mkdir` through a subprocess. I'm not sure this would work correctly as is.

Reply:
👍 for `pathlib` instead of `shutil`

Reply:
Thanks guys - this is what I needed 👍

Reply:
Oh, one thing to note: I don't think `pathlib` has a method to remove a directory containing files. IIRC `Path.rmdir` will only remove an empty directory. You may have to use `shutil.rmtree` if the directory has stuff in.

Reply:
You might also want to look at mkdirs() as this will create any intermediate dirs rather than bailing out (like mkdir() would)

Reply:
Thanks Mark, I think if I can I will use python because it means I can test it more easily but that really interesting to know about `mkdirs()`.